lib/facter/easyrsa.rb

```diff
@@ -0,0 +1,57 @@
+Facter.add(:easyrsa) do
+  confine :kernel => 'Linux'
+  setcode do
+    version = ''
+    binaryv2 = ''
+    binaryv3 = ''
+    operatingsystem = Facter.value(:operatingsystem)
+    operatingsystemrelease = Facter.value(:operatingsystemrelease)
+
+    case operatingsystem
+    when /RedHat|CentOS/
+      binaryv2 = '/usr/share/easy-rsa/2.0/pkitool'
+      binaryv3 = '/usr/share/easy-rsa/3/easyrsa'
+    when /Ubuntu|Debian/
+      case operatingsystemrelease
+      when /8|9|16.04|18.04/
+        binaryv2 = '/usr/share/easy-rsa/pkitool'
+        binaryv3 = '/usr/share/easy-rsa/easyrsa'
+      else
+        binaryv2 = '/usr/share/doc/openvpn/examples/easy-rsa/2.0/pkitool'
+        binaryv3 = '/usr/share/doc/openvpn/examples/easy-rsa/3.0/easyrsa'
+      end
+    when /Archlinux/
+      binaryv2 = '/usr/share/easy-rsa/2.0/pkitool'
+      binaryv3 = '/usr/share/easy-rsa/3/easyrsa'
```

Comment:
`/usr/bin/easyrsa` is the correct path

Reply:
For any version of easyrsa ?

Reply:
for v3. v2 does not exist on Arch Linux.